src/util/time_lua.inl

```diff
@@ -374,6 +374,28 @@ namespace {
     }
 }
 
+/**
+ * Returns the number of seconds between the provided start time and end time. If the end
+ * time is before the start time, the return value is negative. If the start time is equal
+ * to the end time, the return value is 0.
+ * Both start and end times must be valid ISO 8601 date strings.
+ */
+[[codegen::luawrap]] double duration(std::string start, std::string end) {
+    using namespace openspace;
+
+    const double tStart = Time::convertTime(start);
+    const double tEnd = Time::convertTime(end);
+    return tEnd - tStart;
+}
+
+/**
+ * Returns the number of seconds in a Julian year, which is equal to 31557600.
+ */
+[[codegen::luawrap]] double secondsPerYear() {
+    // We could use a call to SPICE here, but the value is a constant anyway
+    return 31557600.0;
```

Comment:
Shouldn't we just use the already defined constant [here](https://github.com/OpenSpace/OpenSpace/blob/55e196e351b443007cc15aad7203686e85b7ff0e/include/openspace/util/timeconversion.h#L37) instead of repeating the magic number?